Fix/Implement Pause On Exceptions (#89)
microsoft/vscode-mono-debug: CHANGELOG.md

## 0.16.3

> * Added support for configuring handled and unhandled exceptions. Configure it on Breakpoints window.

Reviewer: ```suggestion
* Added support for configuring handled and unhandled exceptions in the Breakpoints window.
```
